### DISCUSSION TITLE
job.progress is not a function

### DISCUSSION THREAD

**Discussion by EverCabarcas** [taskforcesh/bullmq#408] [Q&A] [answered]
Hi guys I have the following error in my JS `job.progress is not a function` when I'm trying to set the progress of my process. I'm using a `Sandboxed processors` for my task, this is a little example of my processor code:
``` 
module.exports = async (job, done) => {
    const jobData = job.data.data
    const user = job.data.user

    job.progress(10)
    
    return 'Task successful'
  
  };
````

![image](https://user-images.githubusercontent.com/25714208/108563799-17068780-72d0-11eb-85b2-17774792a15b.png)

I will be really grateful if you guys can help me!! 

Greetings to all of you

**Comment by manast** [comment]
it is ```job.updateProgress```

**Comment by manast** [comment, marked as answer]
https://docs.bullmq.io/guide/workers

**Reply by EverCabarcas** [reply]
Awesome, Thanx @manast I really appreciate your fast response 💪
